src/stdlib/base.bpf.c

```diff
@@ -0,0 +1,14 @@
+#define __KERNEL__
+#include <linux/types.h>
+#include <stddef.h>
+
+#include <bpf/bpf_helpers.h>
+
+extern _Bool bpf_session_is_return(void) __ksym __weak;
+
+_Bool __session_is_return() {
+    if (bpf_session_is_return) {
+        return bpf_session_is_return();
```

Comment:
Neat. Do I understand correctly that the condition will be false if the kfunc doesn't exist?

Reply:
Well... I'm assuming (maybe incorrectly) that this kfunc matches up with the ability to have session probes so if kprobe session feature is not enabled then this function call is not generated:
- https://github.com/bpftrace/bpftrace/blob/master/src/ast/passes/probe_expansion.cpp#L191